README.md

```diff
@@ -45,6 +45,16 @@ Configuration
 
 Copy `config/app.config.template` to `config/app.config` and fill in the required info there.
 
+Set up these environmental variables in `config/app.config` file.
+```
+Service: API Key for accessing Media Cloud (get this on your profile page)
+
+MEDIA_CLOUD_API_KEY = YOUR_MEDIACLOUD_API_KEY_HERE
+
+Service: URL to mediacloud api (Media Cloud backend)
+
+MEDIA_CLOUD_API_URL = YOUR_MEDIACLOUD_API_URL
+```
```

Comment:
I don't think we need to repeat what's in the config/ap.config here. Something like 👇🏽  should do it (if I got what you mean).

```suggestion
**NOTE**: `MEDIA_CLOUD_API_KEY` and `MEDIA_CLOUD_API_URL` are **required**. You can find your `MEDIA_CLOUD_API_KEY` on your profile page.
```